datus/storage/ext_knowledge/store.py

```
@@ -157,3 +155,118 @@ def search_all_knowledge(
+                name = terminology
+
+            # Store the knowledge entry
+            self.store.store_knowledge(
+                subject_path=subject_path,
+                name=name,
+                terminology=terminology,
+                explanation=explanation,
+            )
+
+        except Exception as e:
+            logger.error(f"Failed to add knowledge: {str(e)}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)